### DISCUSSION TITLE
Any Timeline compatibility changes expected? Bonus portrait subsystem question.

### DISCUSSION THREAD

**Discussion by Maican** [dialogic-godot/dialogic#1329] [Q&A]
Hello,

I recently ported my RPG into Godot 4.0 betas and it's been great working with it. Have the Dialogic 2 alpha 4 in my project currently and I was able to convert my old Dialogic 1 timelines and characters.

- Obviously you don't have a crystal ball, but do any of the contributors believe any big timeline changes are in the works, or am I okay to start creating NPC timelines in Alpha 4 without fear of them requiring a bunch of compatibility changes down the line?

- Also does someone know of a way to turn portraits off in Dialogic 2 somewhere in the code with only a couple changes? I assume it will be made a setting to turn off the subsystem in the future, but so far I've just been pulling the most recent code and fixing breaks until it starts running again.

**Comment by Jowan-Spooner** [comment]
Hi @Maican, thanks first of all for trying out 2.x

- It's hard to say how many changes will be made to the syntax. While there are currently no big plans to change it, dialogic 2 is in alpha just because we still consider breaking changes like this a possibility. So if you don't want to do any updating, then don't use it yet. It's hard to tell what aspects might change before we decide to go into beta/release state.
- What exactly do you want from disabling portraits? If you don't join characters, then non should be shown in game. Do you just want to not see the portrait picker in the text node? It shouldn't be shown if you don't add portraits to your characters. If you absolutely do not want the portraits system for whatever reason then maybe create a feature request for that. In theory you could just delete the Character Event folder but I haven't tried it and that change wouldn't survive updates of the plugin.

Hope this helps ;)

**Reply by Maican** [reply]
Thanks that clears everything up, I think I'll go forward making a couple prototype timelines and just not go super hard at the moment!

As for the portraits, I just was playing around with the speech bubble example node, but a bunch of errors were popping up since I didn't have portraits assigned to some of my characters. I am betting it was due to the fact that my old timelines did have portraits and they were ported expecting them. I will try removing the characters joining from the timelines and that would definitely be sufficient for my purposes!
